packages/components/src/components/primitives/Select/Select.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { renderWithProviders } from 'tests/renderWithProviders';
+import { ComposerProvider } from 'theme';
+import { Select } from '.';
+
+describe('Select', () => {
+  test('it should render', () => {
+    renderWithProviders(<Select />);
```

Comment:
can we assert that the component is rendered?

Reply:
Done!